tutorials/plugins/editor/inspector_plugins.rst

```diff
@@ -243,13 +245,14 @@ followed by ``set_bottom_editor()`` to position it below the name.
     func refresh_control_text():
         property_control.text = "Value: " + str(current_value)
 
- .. code-tab:: none C#
```

Comment:
If you take a look at the FIXME above this example, it seems syntax highlighting was disabled purposely:

> .. FIXME: The second tab has the C# lexer for hightlighting disabled for now, as the provided code causes errors.

I'm not sure what errors the message is referring to, but if it's working fine now we can remove the FIXME notice.

Reply:
I'm not getting any error while building it and the generated site looks normal. I removed the FIXME, hopefully it is really fixed and I'm not blind again.

Reply:
It seems the FIXME was added in https://github.com/godotengine/godot-docs/pull/6319.
@mhilbrunner Do you remember why the syntax highlighting was disabled in this example? Is it OK to re-enable it?

Reply:
I bet it's because of the string interpolation syntax. Remember I complained about it on RocketChat? Locally I was seeing errors because of this. Though on CI it seems to run through these code samples perfectly fine.

Reply:
As mentioned on RocketChat, I reverted the change, the problem can be fixed by a follow up PR.